ntoskrnl/ex/sysinfo.c

```diff
@@ -11,12 +11,18 @@
 /* INCLUDES *****************************************************************/
 
 #include <ntoskrnl.h>
+#include <wmidata.h>
+#include <wmistr.h>
 #define NDEBUG
 #include <debug.h>
 
 /* The maximum size of an environment value (in bytes) */
 #define MAX_ENVVAL_SIZE 1024
 
+#define SIG_ACPI 0x41435049
+#define SIG_FIRM 0x4649524D
+#define SIG_RSMB 0x52534D42
```

Comment:
Would 'BMSR' be somewhat more readable/searchable?

Reply:
> Would 'BMSR' be somewhat more readable/searchable?

No.